ui/perfherder/alerts/Pagination.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Row, Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+
+class Paging extends React.Component {
+  constructor(props) {
+    super(props);
+    this.validated = this.props.validated;
+    this.state = {
+      page: this.validated.page ? parseInt(this.validated.page, 10) : 1,
+      count: 0,
+    };
+  }
+
+  navigatePage = page => {
+    this.setState({ page }, this.fetchAlertSummaries);
+    this.props.validated.updateParams({ page });
+  };
+
+  render() {
+    const { pageNums } = this.props;
+    const { page, count } = this.state;
+    return (
+      <div>
+        {pageNums.length > 1 && (
+          <Row className="justify-content-center pb-5">
+            {/* The first and last pagination navigation links
```

Comment:
Lines 23 - 25 are only relevant to the `AlertsView` file so they should not be in this component.